src/text/skeleton.scss

```diff
@@ -3,23 +3,29 @@
 @import '../style/index';
 
 @mixin iui-text-skeleton {
-  display: inline-block;
+  width: fit-content;
   user-select: none;
   color: transparent;
   border-radius: $iui-border-radius;
+  cursor: progress;
```

Comment:
Any reason we shouldn't use `wait` here?

Reply:
`wait` probably makes more sense, I will change it tomorrow when I sign on.

Reply:
Using `wait` cursor now.